swagger_spec_validator/validator20.py

```diff
@@ -57,7 +57,9 @@ def validate_spec_url(spec_url):
     return validate_spec(load_json(spec_url), spec_url)
 
 
-def validate_spec(spec_dict, spec_url='', http_handlers=None):
+def validate_spec(spec_dict, spec_url='', http_handlers=None,
```

Comment:
Maybe it makes more sense to provide a higher-level abstraction? Several possibilities come to mind, e.g. either passing in an arbitrary filesystem path, a file object from which we read or just the schema as JSON string. I wouldn't expose the internal `python package` + `relative_path` way of loading it as our API.